WordPress/gutenberg
# Apply keyboard shortcut conversions between paragraphs and headings to all editor instances

Follow-up on: #44681

## What?
This PR adds conversion keyboard shortcuts between paragraphs and headings to all editors, in addition to the post editor.

## How?

In addition to adding shortcuts, the following actions have been taken:

- Add E2E test
- Fix a problem with text alignment not being carried over

## Testing Instructions

In the post/edit-site/edit-widgets/customize-widgets, check the following:

- Should convert from a paragraph to a heading with Ctrl + Alt (Option) + [1-6]
- Should convert from a heading to a paragraph with Ctrl + Alt (Option) + 0
- Should change heading level with Ctrl + Alt (Option) + [1-6]
- Text alignment attribute should be inherited
- Two new items have been added to the keyboard shortcut modal
